### DISCUSSION TITLE
How do I properly space "flex-*-reverse" items with "space-*-*" utilities?

### DISCUSSION THREAD

**Discussion by ariesclark** [tailwindlabs/tailwindcss#3078] [Help] [answered]
### Describe the problem:
`space-x-*` used with `flex-row-reverse` doesn't properly space as you'd expect it would.

### Link to a minimal reproduction:
https://play.tailwindcss.com/VDTZ7rAdIu

**Comment by Krutsch** [comment, marked as answer]
You also need to inverse the spacing in this case:
Try adding `space-x-reverse`

**Reply by simonswiss** [reply]
That's it 👍

Here's a video that covers this topic, if interested:

https://www.youtube.com/watch?v=wTRTC4JNSGs

**Comment by simonswiss** [comment]
That's it - @Krutsch's answer is correct.

I will convert this to a GitHub discussion as some folks might find this one useful 👍
